version_info

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
-VM_IMPORT_CONTROLLER_CHART_VERSION="1.5.0-dev.0"
-VM_IMPORT_CONTROLLER_IMAGE="rancher/harvester-vm-import-controller:v1.5.0-dev.0"
-PCIDEVICES_CONTROLLER_CHART_VERSION="1.5.0-dev.1"
-PCIDEVICES_CONTROLLER_IMAGE="rancher/harvester-pcidevices:v1.5.0-dev.1"
-HARVESTER_SEEDER_CHART_VERSION="1.5.0-dev.0"
-HARVESTER_SEEDER_IMAGE="rancher/harvester-seeder:v1.5.0-dev.0"
+VM_IMPORT_CONTROLLER_CHART_VERSION="1.5.0-rc2"
+VM_IMPORT_CONTROLLER_IMAGE="rancher/harvester-vm-import-controller:v1.5.0-rc2"
+PCIDEVICES_CONTROLLER_CHART_VERSION="1.5.0-rc2"
+PCIDEVICES_CONTROLLER_IMAGE="rancher/harvester-pcidevices:v1.5.0-rc2"
+HARVESTER_SEEDER_CHART_VERSION="1.5.0-rc2"
+HARVESTER_SEEDER_IMAGE="rancher/harvester-seeder:v1.5.0-rc2"
 NVIDIA_DRIVER_RUNTIME_CHART_VERSION="1.5.0-dev.0"
```

Comment:
Why the driver version doesn't change?

BTW, in the future, if we need an OOB for any embedded addon charts, we may add an incremental number just after the release version, for example, 1.5.0-1 (1.5.0-[1-9])

cc @harvester/release @harvester/tpm

Reply:
>Why the driver version doesn't change?

will be fixed in rc3.